### ISSUE TITLE
Status of CanMV230 2.0 release?

### ISSUE THREAD

**Issue by kelas** [kendryte/k230_sdk#69]
### Feature description

Earlier this year there was a clear mention of a 2.0 board, "scheduled to be released in May":

![image](https://github.com/user-attachments/assets/f04a2b87-39b8-494a-81be-0f60e2e4f707)

### Want resolve what problem

Not that we are in August, are there any news on 2.0? Some vendors are reporting that the sales of 1.1 board dropped sharply in the past few months, which can be attributed to the announcement of 2.0, which seems to be lagging.

What's the status then?

### Anything else

The main promise of 2.0, as I understand, is native 4-lane CSI support, which is something a lot of folks would be interested in, myself included.

**Comment by kelas** [comment]
Hey everyone,

A quick heads up: a board very similar to 2.0 layout appears to be available for sale from 01studio and features a faster and larger RAM chip (see below) but:

* Still no amalgamated 4-lane CSI port, which is kinda lame
* The price tag is a bit over the top
* I wouldn't expect an easy ride making it work with official SDK

Any thoughts on that?

![image](https://github.com/user-attachments/assets/8750c768-a6fc-4ca7-8b52-3b3e85655644)

**Comment by huangzhenming** [comment]
Yes, you got it. This board was designed base on V2.0. 

Still no amalgamated 4-lane CSI port, which is kinda lame
=> There is still no amalgamated 4-lane CSI port for 01Studio board, which is a bit disappointing. However, you can combine CSI0 and CSI1 into a 4-lane CSI port, which is much easier by building your own converter board compared to v1.1.
The price tag is a bit over the top
=> No comment. As i know they changed the onboard sensor to GC2093, its performance is far better than previous OV5647,  and the DDR size is doubled.
I wouldn't expect an easy ride making it work with official SDK
=> SDK V1.7 is supposed to support this board which will be released by end of this Month. You can build it with "make CONF=k230_canmv_01studio_defconfig“

**Comment by CaptainJackey** [comment]
> Hey everyone,
> 
> A quick heads up: a board very similar to 2.0 layout appears to be available for sale from 01studio and features a faster and larger RAM chip (see below) but:
> 
> * Still no amalgamated 4-lane CSI port, which is kinda lame
> * The price tag is a bit over the top
> * I wouldn't expect an easy ride making it work with official SDK
> 
> Any thoughts on that?
> 
> ![image](https://private-user-images.githubusercontent.com/1530846/356996954-8750c768-a6fc-4ca7-8b52-3b3e85655644.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxNzM5MTcsIm5iZiI6MTcyNTE3MzYxNywicGF0aCI6Ii8xNTMwODQ2LzM1Njk5Njk1NC04NzUwYzc2OC1hNmZjLTRjYTctOGI1Mi0zYjNlODU2NTU2NDQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MDFUMDY1MzM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTRlNGEwYzhjZDVjZTk0ZWViNjA5ZmY5MzdhYWI0NTc3NzUxZTg0ZDBiYWRjOWM1NWU0ZjljYmNkZTA0OTczMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QWrhh_lkbz7Ug06-nDZiFQ8i42r59NBSFe3sONRnmjw)

01Studio CanMV K230 Start selling : https://www.aliexpress.com/item/1005007573129445.html 
01Studio CanMV K230 Tutorial：https://wiki.01studio.cc/docs/canmv_k230（Please use Google Translate）

**Comment by kelas** [comment]
@CaptainJackey @huangzhenming 

Thank you for your inputs! I've got a 01studio board, and so far I find it to be excellent.

Before I close the thread, one more question: can you give a clue on whether the following SOM (or at least its native motherboard) is also going to enjoy official SDK support? 

https://www.aliexpress.com/item/1005007287343051.html

The SOM form factor (and the price tag) fit my requirements much closer, but the vendor of the board seems to capitalize on Python, which is not what I'm looking for (I'd very much prefer to stick to official C SDK).

Are there plans to support this one as well?

**Comment by huangzhenming** [comment]
Yes, the latest version (V1.7) of the K230 SDK supports the 01Studio board. However, I'm unable to access AliExpress (as it blocks visitors from China), so I can't check the exact board you're referring to. I believe it can be easily adapted with minor modifications to the SDK source code.

**Comment by kelas** [comment]
@huangzhenming here's what I'm talking about:

![image](https://github.com/user-attachments/assets/b4fb1502-5a8c-4dc6-b06d-dd0b6eb2bba1)

**Comment by Rongkangqi** [comment]
> @huangzhenming here's what I'm talking about:
> 
> ![image](https://private-user-images.githubusercontent.com/1530846/369045978-b4fb1502-5a8c-4dc6-b06d-dd0b6eb2bba1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4MjQ4MTksIm5iZiI6MTcyNjgyNDUxOSwicGF0aCI6Ii8xNTMwODQ2LzM2OTA0NTk3OC1iNGZiMTUwMi01YThjLTRkYzYtYjA2ZC1kZDBiNmViMmJiYTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MjBUMDkyODM5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDVlZDRhMzU4MTcwOGE0ODNlODhhMWJlZmU2NDQwMWU3MmRhMWYzNmRhZTIzYmIwYzdjNjVhMzE2ZDA1NWVmMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.uWbu0Pwn7IH4AaxNoi9RzVvYYdKsz_We_RJdhfOhNNo)

hello,kelas. As for the K230 core board, since we have modified a lot of content based on the K230 SDK, we temporarily maintain it on Tencent's coding in China, and the maintained version is V1.6. You can directly execute:
git clone https://e.coding.net/weidongshan/dshanpi-canmv/k230_sdk.git

K230 core board related tutorials: [https://eai.100ask.net/CanaanK230/Userdoc](https://eai.100ask.net/CanaanK230/Userdoc)

Get the latest SDK based on the K230 core board.
We will incorporate our differentiated parts into the K230 SDK during this time, so please rest assured

**Comment by kelas** [comment]
Yaaay!

Great news, fine folks at Banana probably heard my repeated woes about lack of 4-lane port on all available k230 boards, so here we go (can't wait for this sweetheart to hit the shelves). CSI0+CSI1, voila!

(All in all, great to see that k230 is picking up the pace so fast. Awesome system with big future.)

![image](https://github.com/user-attachments/assets/641bca57-bb70-4729-acc4-c7765d2c092d)
